web_development_101/the_basics/introduction_to_web_development.md

```diff
@@ -1,50 +1,49 @@
+Earlier, we mentioned that web development work could be in the front end, the back end, or the full stack. What exactly do these terms mean?
 
-The "Front End" is the stuff you see on the website in your browser, including the presentation of content and the user interface elements like the navigation bar.
-Front End Developers make use of HTML, CSS, Javascript, and their relevant frameworks to ensure that content is presented effectively and that users have an excellent experience.
+The **front end** is the stuff you see on the website in your browser, including the presentation of content and user interface elements like the navigation bar.
+Front-end developers use HTML, CSS, JavaScript, and their relevant frameworks to ensure that content is presented effectively and that users have an excellent experience.
 
-The "Back End" refers to the guts of the application, which live on the server. They manipulate data appropriately to make sure the Front End has what it needs. This can become very complicated as things scale.
-Back End Developers make use of programming languages like Java, Python, and Ruby (among many others) to work with data.
+The **back end** refers to the guts of the application, which live on the server. The back end manipulates data to ensure that the front end has what it needs, which can become very complicated as things scale.
+Back-end developers use programming languages like Java, Python, and Ruby to work with data.
 
-"Full Stack" Developers marry both sides into one. A Full Stack developer can comfortably work with both the front and back ends. This is what we focus on building you up to here.
+**Full-stack developers** are comfortable working with both the front and back ends. In The Odin Project, we focus on teaching you full-stack development, covering all aspects of web development.
 
-For more details, Udacity has a great blog post on it: [3 Web Dev Careers Decoded: Front-End vs Back-End vs Full Stack](http://blog.udacity.com/2014/12/front-end-vs-back-end-vs-full-stack-web-developers.html)
+For more detail, Udacity has a great blog post on this topic: [3 Web Dev Careers Decoded: Front-End vs Back-End vs Full Stack](http://blog.udacity.com/2014/12/front-end-vs-back-end-vs-full-stack-web-developers.html)
 
 ### Types of Careers
 
-Now that you know about the different types of web developers, let's cover what we mentioned earlier about the different types of clients/employers you may work with.
+Now that you know about the different types of web developers, let's cover what we mentioned earlier about the different types of clients and employers you may work with.
 
-There are large tech companies, like Google, Facebook, Amazon, etc. They have very high hiring requirements, but they offer excellent pay, benefits, and opportunities.
+Large tech companies, such as Google, Facebook, and Amazon, have very stringent hiring requirements, but they offer excellent pay, benefits, and opportunities.
```

Comment:
Love "stringent" here 😍

Reply:
That's a very long sentence with a lot of commas. I think it needs to be broken up and structured better?